docs/src/features/atlases.md

```diff
@@ -1,21 +1,34 @@
-They provide a basic connection between manifolds and Euclidean spaces.
+Atlases on an ``n``-dimensional manifold $\mathcal M$ are collections of charts ``\mathcal A = \{(U_i, \varphi_i) \colon i \in I\}``, where ``I`` is a (finite or infinte) index family, such that ``U_i \subseteq \mathcal M`` is an open set and each chart ``\varphi_i: U_i \to \mathbb{R}^n`` is a homeomorphism. This means, that ``\varphi_i`` is bijecive – sometimes also called one-to-one and onto - and continuous, and its inverse ``\varphi_i^{-1}`` is continuous as well.
+The inverse ``\varphi_i^{-1}`` is called (local) parametrization.
+For an atlas ``\mathcal A`` we further require that
+
+```math
+\displaystyle\bigcup_{i\in I} U_i = \mathcal M.
+```
+
+We say ``\varphi_i`` is a chart about ``p``, if ``p\in U_i``
+An atlas provides a connection between a manifold and the Euclidean space ``\mathbb{R}^n``, since
+locally, a chart about ``p`` can be used to identify its neighborhood (as long as you sta in ``U_i``) “looks like” the Euclidean space.
+Most manifolds we consider are smooth, i.e. any change of charts ``\varphi_i\circ\varphi_j^{-1}: \mathbb{R}^n\to\mathbb{R}^n``, ``i,j\in I``, is a smooth function. These change of charts are also called transition maps
 
 Most operations on manifolds in `Manifolds.jl` avoid operating in a chart through appropriate embeddings and formulas derived for particular manifolds, though atlases provide the most general way of working with manifolds.
+Compared to these approaches, using an atlas ist often more technical and time-consuming.
 They are extensively used in metric-related functions on [`MetricManifold`](@ref Main.Manifolds.MetricManifold)s.
 
 Atlases are represented by objects of subtypes of [`AbstractAtlas`](@ref Main.Manifolds.AbstractAtlas).
 There are no type restrictions for indices of charts in atlases.
 
 Operations using atlases and charts are available through the following functions:
 
-* [`get_chart_index`](@ref Main.Manifolds.get_chart_index) can be used to select an appropriate chart for the neighborhood of a given point.
-* [`get_point_coordinates`](@ref Main.Manifolds.get_point_coordinates) converts a point to its coordinates in a chart.
-* [`get_point`](@ref Main.Manifolds.get_point) converts coordinates in a chart to the point that corresponds to them.
-* [`induced_basis`](@ref Main.Manifolds.induced_basis) returns a basis of a given vector space at a point induced by a chart.
-* [`transition_map`](@ref Main.Manifolds.transition_map) converts coordinates of a point between two charts.
+* [`get_chart_index`](@ref Main.Manifolds.get_chart_index) can be used to select an appropriate chart for the neighborhood of a given point ``p``. This function shoudl work deterministic, i.e. for a fixed ``p`` always return the same chart.
+* [`get_parameters`](@ref Main.Manifolds.get_parameters) converts a point to its local coordinates, also called parameters with respect to the chart in a chart.
```

Comment:
Should we keep using both words (coordinates and parameters) to refer to parameters in a chart? I'd prefer sticking with one term, i.e.  "parameters". If you want to mention coordinates here, could you phrase it more like 'some authors refer to parameters as "local coordinates"' ?

Reply:
Currently I used parameters (and local coordinates in brackets), but maybe we should only do that here and just once in the function docs. So sure, we can do as you propose.

In general, when writing this, I felt quite good with the clear distinction parameters vs coordinates, especially in the RetractionAtlas docs.